cupyx/fallback_mode/fallback.py

```diff
@@ -76,12 +115,248 @@ def __call__(self, *args, **kwargs):
             raise TypeError("'{}' object is not callable".format(
                 type(self._numpy_object).__name__))
 
+        # if ndarray method
+        if self._fallback_array is not None:
+            args = ((self._fallback_array,) + args)
+
         # Execute cupy method
         if self._cupy_object is not None:
-            return utils._call_cupy(self._cupy_object, args, kwargs)
+            return _call_cupy(self._cupy_object, args, kwargs)
 
         # Execute numpy method
-        return utils._call_numpy(self._numpy_object, args, kwargs)
+        return _call_numpy(self._numpy_object, args, kwargs)
 
 
 numpy = _RecursiveAttr(np, cp)
+
+# -----------------------------------------------------------------------------
+# utils
+# -----------------------------------------------------------------------------
+
+
+def _call_cupy(func, args, kwargs):
```

Comment:
These function definitions (`_call_cupy` and `_call_numpy`) seem better to be moved after `_get_fallback_result`.